fix: security vulnerabilities using ZAP

sanjay-k1910 opened the pull request:
## What?
- Resolved security vulnerabilities of ZAP

ajile-in commented on src/app/LayoutCommon.astro:
```diff
@@ -17,22 +18,63 @@ envKeys.forEach((item) => {
+
+// Astro.response.headers.set('Content-Security-Policy',"script-src 'self'; frame-ancestors 'self'");
+Astro.response.headers.set('X-Frame-Options', "SAMEORIGIN");
+Astro.response.headers.set('X-CREDEBL',"SSI");
```
Is this line still needed? I think we can remove it.

sanjay-k1910 replied:
Removed the unused code.

ajile-in commented on src/middleware.ts:
```diff
@@ -0,0 +1,58 @@
+// export default async (request: Request, context: any) => {
```
Please remove all the commented and unused code.

sanjay-k1910 replied:
Yes Sir, I'm using this PR to test the security-related changes on deployed environments.
This PR is not yet in the merge status.

ajile-in reviewed (changes requested):
Finally, which approach is working? `deployctl.yaml`? or something else?

sanjay-k1910 commented:
> Finally, which approach is working? `deployctl.yaml`? or something else?

Removed the `deployctl.yaml` file and implemented headers in `middleware.ts` file.

tipusinghaw reviewed (approved):
LGTM
